### ISSUE TITLE
add per_page parameter to song_annotations

### ISSUE THREAD

**Issue by marilenadaquino** [johnwmillr/LyricsGenius#245]
TLDR: `song_annotations` returns only 10 results, I would like to get all annotations for a song. 

I'm performing a text search on all Genius. I get a list of songs that somehow reference the term and I'd like to get alla annotations for these songs (since the term I searched is likely to be there). But the method  (https://lyricsgenius.readthedocs.io/en/master/reference/genius.html#lyricsgenius.Genius.song_annotations) returns only 10 results, which 99% of times do not include that very first term I've searched. 

I'd like a parameter to specify how many results to return - and ideally also one for the pagination of results. 

I could not find any docs on Genius how to do this with their API (I cannot even get the 10-result list from that API)
Any help would be appreciated! Thank you in advance
